src/system/Form/Label.tsx

```diff
@@ -18,8 +17,16 @@ export const baseLabelStyle = {
 	lineHeight: 1.5,
 	color: baseLabelColor,
 };
+interface LabelProps {
```

Comment:
could extends `BoxProps` here and inherit many of these props

Reply:
Since you are passing to `Box`, I meant a pure `extends` like

`interface LabelProps extends BoxProps {`

But if there are reasons not to do it, you can ignore this, just a tiny suggestion

Reply:
done in https://github.com/Automattic/vip-design-system/pull/251/commits/0ce6d477c37878926efcd44f92a91f9a4cc654f1